localstack/cli/localstack.py

```diff
@@ -180,12 +180,19 @@ def cmd_stop():
 )
 @publish_invocation
 def cmd_logs(follow: bool):
+    from localstack.utils.bootstrap import LocalstackContainer
     from localstack.utils.docker_utils import DOCKER_CLIENT
 
     container_name = config.MAIN_CONTAINER_NAME
+    logfile = LocalstackContainer(container_name).logfile
 
     if not DOCKER_CLIENT.is_container_running(container_name):
         console.print("localstack container not running")
+        console.print("printing logs from previous run")
+        if os.path.exists(logfile):
+            with open(logfile) as fd:
+                for line in fd:
+                    console.print(line.rstrip("\n\r"))
```

Comment:
we probably shouldn't be using `console.print`, since it uses rich to print colors. maybe `click.echo` would be more appropriate. and we can omit the newline strip by simply instructing click not to output a newline.

```suggestion
                    click.echo(line, nl=False)
```